core/lib/spree/core/controller_helpers/order.rb

```diff
@@ -21,10 +21,10 @@ def simple_current_order
           @simple_current_order = find_order_by_token_or_user
 
           if @simple_current_order
-            @simple_current_order.last_ip_address = ip_address
+            @simple_current_order.update(last_ip_address: ip_address)
```

Comment:
One though before merging: should this be `update!`? We don't check the response so it probably should.